.github/workflows/cpu_test.yml

```diff
@@ -15,12 +15,9 @@ jobs:
       matrix:
         python-version: ["3.10", "3.11"]
     container:
-      image: us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:nightly_${{ matrix.python-version }}_tpuvm_cxx11_20250307
+      image: us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:nightly_${{ matrix.python-version }}_tpuvm_cxx11_20250114
```

Comment:
can we make the pin date explicitely called out as config?

Reply:
https://github.com/AI-Hypercomputer/torchprime/issues/153